api/Navigator.json

```diff
@@ -1266,7 +1264,7 @@
               "version_added": false
             },
             "webview_android": {
-              "version_added": false
+              "version_added": "68"
```

Comment:
Are you sure about this? Chrome for Android says false.

Reply:
Just checked for support in Chrome Android, and it looks like there actually _is_ support.  I'll update the Chrome/Opera Android and Samsung Internet data in a follow-up PR.

Reply:
Okay, sounds good!